app/presenters/queries/content_item_presenter.rb

```diff
@@ -143,6 +145,11 @@ def parse_results(results)
             int_columns.each { |c| parse_int_column(result, c) }
             parse_state_history(result)
 
+            if self.include_warnings
+              warnings = get_warnings(result)
+              result["warnings"] = warnings unless warnings.nil?
```

Comment:
I've changed this around now as a result of adding warnings in regardless of there being any.